dashboard/client/public/index.html

```diff
@@ -11,7 +11,7 @@
   <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico?v=1">
   <title>OpenFaaS Cloud</title>
   <!-- inject global variables from the server side -->
-  <script>
+  <script id="server_data" data-signed-in="__IS_SIGNED_IN__">
     window.BASE_HREF = '__BASE_HREF__';
     window.PUBLIC_URL = '__PUBLIC_URL__';
```

Comment:
Pleas move `__IS_SIGNED_IN__ ` to `window` for consistency. Then propose moving all of the elements to a HTML tag later.